newrelic/common/package_version_utils.py

```diff
@@ -12,12 +12,52 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# To use:
```

Comment:
This is great! Instead of putting it here though, can we move this into the function as a doc string:
```
def get_package_version(name):
    """Gets the version of the library.
    :param name: The name of library.
    :type name: str
    :return: The version of the library. Returns None if can't determine version.
    :type return: str or None

    Usage::
        >>> get_package_version("botocore")
                "1.1.0"
    """
```

Reply:
Good call!  Done